handbook/engineering/observability/monitoring_architecture.md

```diff
@@ -109,7 +109,7 @@ There is currently no process defining how custom additions should be made to ou
 
 ## Sourcegraph Cloud
 
-This section describes how our monitoring stack is used in Sourcegraph Cloud and what customizations we have made.
+[Sourcegraph Cloud](../deployments/instances.md#sourcegraph-com) offers all the monitoring capabilities of a [standard Sourcegraph deployment](#standard-sourcgraph-deployments), as well as the following customizations and specific usages.
```

Comment:
What following customizations and specific usages? Missing?

Reply:
Oh there's more on this page: Sourcegraph Cloud customizations include some alerting changes, blackbox exporter, cloudflare etc https://about.sourcegraph.com/handbook/engineering/observability/monitoring_architecture#sourcegraph-cloud this was just a slight rewording while i was on the page